From 27267616896168fe1177c42a73246a8a9c551be3 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 11 Apr 2013 11:07:31 +0200
Subject: Change qemu to qemu-kvm

We use qemu-kvm as name for RHEL 7. This patch fixes various pathes and files that uses qemu so they use qemu-kvm instead.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

diff --git a/os-posix.c b/os-posix.c
index d39261d..3a4678a 100644
--- a/os-posix.c
+++ b/os-posix.c
@@ -82,7 +82,7 @@ void os_setup_signal_handling(void)
 /* Find a likely location for support files using the location of the binary.
    For installed binaries this will be "$bindir/../share/qemu".  When
    running from the build tree this will be "$bindir/../pc-bios".  */
-#define SHARE_SUFFIX "/share/qemu"
+#define SHARE_SUFFIX "/share/qemu-kvm"
 #define BUILD_SUFFIX "/pc-bios"
 char *os_find_datadir(const char *argv0)
 {
diff --git a/ui/vnc.c b/ui/vnc.c
index dfc7459..3726352 100644
--- a/ui/vnc.c
+++ b/ui/vnc.c
@@ -3260,7 +3260,7 @@ void vnc_display_open(DisplayState *ds, const char *display, Error **errp)
     }
 
 #ifdef CONFIG_VNC_SASL
-    if ((saslErr = sasl_server_init(NULL, "qemu")) != SASL_OK) {
+    if ((saslErr = sasl_server_init(NULL, "qemu-kvm")) != SASL_OK) {
         error_setg(errp, "Failed to initialize SASL auth: %s",
                    sasl_errstring(saslErr, NULL, NULL));
         goto fail;
